l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -116,26 +116,32 @@ def getPartnerBase(self, DatiAnagrafici):
                 DatiAnagrafici.IdFiscaleIVA.IdPaese,
                 DatiAnagrafici.IdFiscaleIVA.IdCodice
             )
-        partners = partner_model.search([
-            '|',
-            ('vat', '=', vat or 0),
-            ('fiscalcode', '=', cf or 0),
-        ])
-        commercial_partner = False
+        partners = partner_model
```

Comment:
Can I ask why this? would't be better partners= False? Am I missing something?

Reply:
`partner_model` is an empty set. Using `partners= False` , line `len(partners)` would raise exception

Reply:
Agreed, in that case would be possible to check `if partners and len(partners) > 1`. Btw it's ok, thx